manifests/controller-manager/vccm-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vccm
+  namespace: kube-system
+data:
+  1.2.3.4.username: "Replace with output from `echo -n YOUR_VCENTER_USERNAME | base64`"
```

Comment:
Any specific reasons why we should keep the username and password encoded in base64? I believe this was added originally to provide some obfuscation for credentials because they were stored in a configuration file on the host, ideally using k8s secrets should not require this obfuscation anymore.

Reply:
@frapposelli unless I missed the option for it (entirely possible), I thought secrets are still required to be base64 encoded. Using my 1.11.1 cluster, I attempted to create a secret without base64 encoding, this is the error you get:

```
[vonthd@k8master Documents]$ kubectl create -f vccm-secret-nobase64.yaml 
Error from server (BadRequest): error when creating "vccm-secret-nobase64.yaml": Secret in version "v1" cannot be handled as a Secret: v1.Secret.Data: decode base64: illegal base64 data at input byte 5, error found in #10 byte of ...|"XXXXX","10.160.2|..., bigger context ...|:"v1","data":{"10.160.221.107.password":"XXXXXX","10.160.221.107.username":"administrator@vsphere.|...
```

Reply:
You're absolutely right :)